js/civicrm_stripe.js

```diff
@@ -180,6 +180,27 @@
           }
         }
       }
+
+      // card_number not empty validation
+      if (!$form.find("input#credit_card_number").val()) {
+        $form.find("input#credit_card_number").parent().find('.crm-error').remove();
+        $form.find("input#credit_card_number").parent().append('<span class="crm-error msg">Credit-Card Number is required.</span>');
+        return false;
+      }
+      else {
+        $form.find("input#credit_card_number").parent().find('.crm-error.msg').remove();
+      }
+
+      // cvv not empty validation
+      if (!$form.find("input#cvv2").val()) {
+        $form.find("input#cvv2").parent().find('.crm-error').remove();
+        $form.find("input#cvv2").parent().append('<span class="crm-error msg">CVV Number is required.</span>');
```

Comment:
Multiple uses of `$form.find("input#cvv2")` - why don't save it to a var? What do you think?